proximo/proximo_source.go

```diff
@@ -53,6 +53,7 @@ type asyncMessageSource struct {
 	conn          *grpc.ClientConn
 	consumerGroup string
 	topic         string
+	offset        int64
```

Comment:
Why int64 instead of a proper type?

(I think it's because that's how it was already, commented out? - which was probably me)

Reply:
happy to fix, although this is private.